charts/logging-operator/templates/pre-delete-hook.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: batch/v1
```

Comment:
I think this job should be opt in:
- uninstall does not mean delete every CRs
- argocd does not support pre-delete hook